docs/PageLabels.md

```diff
@@ -0,0 +1,99 @@
+# Page Labels
```

Comment:
This new documentation page is really clear & well written, good job! 👍

One little thing that was a bit unclear to me while reading the page: are page labels visible, rendered by PDF viewers, or is it just metadata?

Maybe it would be worth clarifying that on this page?

Reply:
My mistake, Adobe Acrobat Reader indeed render page labels:
![image](https://github.com/user-attachments/assets/22444343-9f32-4da8-b9ad-a5e789c1f68b)

It may be worth adding documentation & screenshots, still 🙂